index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <script src='main.js'></script>
+</head>
+
+<body>
+  <div class="main-container">
+    <nav>
+      <div class="logo">
+        <span>Logo</span>
+      </div>
+      <div class="menu">
+        <button class="menu-btn">
+          <img src="assets/Icon-Menu.png" alt="menu button">
+        </button>
+      </div>
+    </nav>
+    <div class="headline">
+      <div>
+        <h1>Hey There.
+          I’m Alex</h1>
+
+        <h2>I’am a Software Developer</h2>
+
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+      </div>
+      <div class="contact-info">
+        <a href="#"> <img src="assets/Icon-GitHub.png" alt="github logo"></a>
+        <a href="#"> <img src="assets/Icon-Linkedin.png" alt="linkedin logo"></a>
+        <a href="#"> <img src="assets/Icon-Angelist.png" alt="angelist logo"></a>
+        <a href="#"> <img src="assets/Icon-Medium.png" alt="medium logo"></a>
+        <a href="#"> <img src="assets/Icon-Twitter.png" alt="twitter logo"></a>
+      </div>
```

Comment:
- [ ] Kindly notice that this social-links list should be wrapped inside a list (`<ul>` tag), (`<li>` tag), and finally the links (`<a>` tags).

_Please look up the comment about Semantic HTML._